plat/mediatek/mt6795/drivers/uart/8250_console.S

```diff
@@ -0,0 +1,112 @@
+/*
```

Comment:
This looks very similar to plat/mediatek/mt8173/drivers/uart/8250_console.S. Can this be re-used?